screens/SettingsScreen.js

```diff
@@ -36,7 +48,9 @@ export default class SettingsScreen extends React.Component {
                             <Right />
                         </Header>
                         <Content>
-                            <Text>Settings here...</Text>
+                            <Button full onPress={this._logout}>
```

Comment:
i just put logout here for now, so you can test the full flow (signing up, then logging out by clicking here, then logging in)